# mpc
receding: 10 # 10
step_time: 0.5
ref_speed: 0.3
device: 'cpu'
time_print: False
collision_threshold: 0.01

# robot
robot:
  kinematics: 'diff'
  max_speed: [3.0, 1.14]
  max_acce: [0.5, 1.14]
  length: 0.322
  width: 0.22

# initial path
ipath:
  interval: 0.2
  waypoints: 
  curve_style: 'line'   # 'dubins' or reeds or line
  min_radius: 0.0
  loop: False
  arrive_threshold: 0.3
  close_threshold: 0.1
  arrive_index_threshold: 3
  
# proximal alternating minimization network
pan:
  iter_num: 2
  dune_max_num: 100
  nrmp_max_num: 10
  dune_checkpoint: None
  iter_threshold: 0.1

# adjust parameters
adjust:
  q_s: 0.7
  p_u: 1.0
  eta: 10.0
  d_max: 0.30
  d_min: 0.25